example/webpack.config.js

```diff
@@ -5,8 +5,10 @@ module.exports = {
     entry: ['./src/index.ts'],
     devtool: "source-map",
     devServer: {
-        contentBase: path.join(__dirname, 'public'),
-        disableHostCheck: true,
+        static: path.join(__dirname, 'public'),
+        allowedHosts: [
```

Comment:
You are right, Bernd. Gitpod doesn't run on localhost, and that is why the `allowedHosts: [0.0.0.0]` would throw an error.
The video I recorded above worked because in there I was running Gitpod on the master branch, not on this PR's branch.
Now I changed `allowedHosts` to `all`.
Also, while investigating I saw that the `ts-loader` package was throwing exceptions with Webpack 5, so I updated it from version 5.2.2 to 9.2.6.